server/CGameHandler.cpp

```diff
@@ -1673,8 +1673,11 @@ void CGameHandler::newTurn()
 				if (monthType < 25)
 				{
 					n.specialWeek = NewTurn::BONUS_GROWTH; //+5
-					std::pair<int, CreatureID> newMonster(54, VLC->creh->pickRandomMonster(getRandomGenerator()));
-					//TODO do not pick neutrals
+					std::pair<int, CreatureID> newMonster(54, 0);
```

Comment:
```suggestion
					std::pair<int, CreatureID> newMonster(54, CreatureID());
```

Reply:
I am not sure if this is proper fix for this issue. That's because neutral faction alignment does not have to mean creatures without town. Creatures from conflux, stronghold and fortress towns have neutral alignment, so for example heroes from these factions can sacrifice both creatures and artifacts for experience etc.